wazuh_vm_tester/src/vm_tester/strategies/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""
+Connection strategies for Wazuh VM Tester.
+"""
+
+from .base import ConnectionStrategy
+from .local import LocalStrategy
+from .ssh import SSHStrategy
+from .ansible import AnsibleStrategy
+from .ami import AMIStrategy
+from .factory import StrategyFactory
+
+__all__ = [
+    "ConnectionStrategy",
+    "LocalStrategy",
+    "SSHStrategy",
+    "AnsibleStrategy",
+    "AMIStrategy",
+    "StrategyFactory",
+]
```

Comment:
add end of line file

Reply:
Done in [0cbff7b](https://github.com/wazuh/wazuh-virtual-machines/pull/266/commits/0cbff7bf4fc34532c19029684c71e9ef28e4253b)